lambda_code/cognito/UserPoolClient/index.py

```diff
@@ -79,22 +92,32 @@ def update(self):
             # We need a new GlobalTable, switch to create and let CLEANUP delete the old one
             raise ValueError("Change of GenerateSecret is not supported for update, please delete and recreate your client")
 
-        params = {'UserPoolId': self.user_pool_id, 'ClientName': self.client_name,
-                       'RefreshTokenValidity': self.refresh_token_validity, 'ReadAttributes': self.read_attributes,
-                       'WriteAttributes': self.write_attributes, 'ExplicitAuthFlows': self.explicit_auth_flows,
-                       'SupportedIdentityProviders': self.supported_identity_providers,
-                       'CallbackURLs': self.callback_urls,
-                       'LogoutURLs': self.logout_urls, 'DefaultRedirectURI': self.default_redirect_uri,
-                       'AllowedOAuthFlows': self.allowed_oauth_flows, 'AllowedOAuthScopes': self.allowed_oauth_scopes,
-                       'AllowedOAuthFlowsUserPoolClient': self.allowed_oauth_flows_user_pool_client}
+        params = {
+            'UserPoolId': self.user_pool_id,
+            'ClientName': self.client_name,
+            'ReadAttributes': self.read_attributes,
+            'WriteAttributes': self.write_attributes,
+            'ExplicitAuthFlows': self.explicit_auth_flows,
+            'SupportedIdentityProviders': self.supported_identity_providers,
+            'CallbackURLs': self.callback_urls,
+            'LogoutURLs': self.logout_urls,
+            'DefaultRedirectURI': self.default_redirect_uri,
+            'AllowedOAuthFlows': self.allowed_oauth_flows,
+            'AllowedOAuthScopes': self.allowed_oauth_scopes,
+            'AllowedOAuthFlowsUserPoolClient': self.allowed_oauth_flows_user_pool_client,
+            'RefreshTokenValidity': self.refresh_token_validity,
+            'AccessTokenValidity': self.access_token_validity,
+            'IdTokenValidity': self.id_token_validity,
+            'TokenValidityUnits': self.token_validity_units,
+        }
         # Remove all params that are None
         params = {k: v for k, v in params.items() if v is not None}
 
         boto_client = self.get_boto3_client('cognito-idp')
 
         response = boto_client.update_user_pool_client(ClientId=self.physical_resource_id,
                                                        **params
-                                                       )
+                                                        )
```

Comment:
Is this auto-formatted? Seems strange it doesn't go
```
response = boto_client.update_user_pool_client(
    ClientId=self.physical_resource_id,
    **params 
)
```

Reply:
No, i added the space manually to satisfy the visual notification of the linter.